commit e5daac418002d851d11f280ac12331c7d6c779c6 (refs/changes/29/2429/3)
Author: Dan McDonald <danmcd@joyent.com>
Date:   2017-09-07T11:33:01-04:00 (2 years, 1 month ago)
    
    docs/index.md text accidentally rendered in-table.
    Reviewed by: Jason King <jason.king@joyent.com>

diff --git a/docs/index.md b/docs/index.md
index 3cc9bf7..18d7c2d 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -90,6 +90,7 @@ update time.
 | zfs_data_compression     | String                        | Specifies a compression algorithm used for the VM's data dataset                                                                                                                                                          | No                  | Yes    | Yes    |
 | zfs_io_priority          | Number                        | ZFS IO Priority                                                                                                                                                                                                           | Yes                 | Yes    | Yes    |
 | zlog_max_size            | Number                        | Sets the maximum size of the stdio.log file for a docker zone before rotation. NOTE: To be used by sdc-docker only.                                                                                                       | No                  | Yes    | Yes    |
+
 Furthermore, when dealing with KVM VMs there are additional attributes to know
 about and are specific to KVM being a different type of virtualization: cpu_type,
 disks, disk_driver, nic_driver and vcpus. KVM VMs require at least
